### ISSUE TITLE
Text is backwards in RTL language basemaps

### ISSUE THREAD

**Issue by gavinr-maps** [Esri/esri-leaflet-vector#204]
### Describe the bug

The [languages sample](https://github.com/Esri/esri-leaflet-vector/blob/5e6aa8cb66a7e9675ce79051d7501cdaf9a5ddce/examples/languages.html) we have in the repo includes right-to-left (RTL) languages like Arabic. When those languages are chosen in the sample, the text is in the wrong direction (it is left-to-right instead of right-to-left).

#### Arabic:

1. Open this sample that uses the **ArcGIS API for JavaScript**: https://jsbin.com/qohubem/2/edit?html,output
    - ![image](https://github.com/Esri/esri-leaflet-vector/assets/137905994/b35c27ae-f8e3-47ef-914e-5aa594659388)

3. In a separate tab, open this sample that uses **Esri Leaflet Vector**: https://jsbin.com/qategug/edit?html,output
    - ![image](https://github.com/Esri/esri-leaflet-vector/assets/137905994/d12522e0-485f-4f0d-9e57-6d40a3f16193)

5. Compare the two maps
    - Expected: The map labels should look the same
    - Actual: the text in the **Esri Leaflet Vector** sample is backwards when compared with the **ArcGIS API for JavaScript** sample.

#### Hebrew:

Same thing for Hebrew, which is another right-to-left language:
1. Open this sample that uses the **ArcGIS API for JavaScript**: https://jsbin.com/nilekem/2/edit?html,output
    - ![image](https://github.com/Esri/esri-leaflet-vector/assets/137905994/e5c84e3d-08cf-41c0-91c8-34a54543da29)

3. In a separate tab, open this sample that uses **Esri Leaflet Vector**: https://jsbin.com/fajowaw/edit?html,output
    - ![image](https://github.com/Esri/esri-leaflet-vector/assets/137905994/ba9bca59-23d8-479f-95fd-b0c4e0e9cc48)
6. Compare the two maps
    - Expected: The map labels should look the same
    - Actual: the text in the **Esri Leaflet Vector** sample is backwards when compared with the **ArcGIS API for JavaScript** sample.

### System Info

```shell
- Leaflet 1.9.4
- Esri Leaflet 3.0.11
- Esri Leaflet Vector 4.1.0
```

### Additional Information

I don't think this is a bug with Esri Leaflet Vector - I think we just need to document how to properly use RTL languages using 
https://github.com/mapbox/mapbox-gl-rtl-text (via https://github.com/maplibre/maplibre-gl-js/issues/220#issuecomment-1048856008). I think once we have an example of properly using that plugin, we must properly document how to use it in our [languages sample](https://github.com/Esri/esri-leaflet-vector/blob/5e6aa8cb66a7e9675ce79051d7501cdaf9a5ddce/examples/languages.html) as well as other places in the doc, like https://developers.arcgis.com/esri-leaflet/maps/change-the-place-label-language/

- https://docs.mapbox.com/mapbox-gl-js/api/properties/#setrtltextplugin

**Comment by gavinr-maps** [comment]
@gowin20 is planning on updating the Esri Leaflet Vector documentation and samples to show how to use the plugin to properly display RTL languages! Thanks @gowin20 :) !

**Comment by gavinr-maps** [comment]
This is resolved in #207 and that PR will be released soon.
